examples/rsp-next-ts/pages/index.tsx

```diff
@@ -0,0 +1,427 @@
+              </ActionButton>
+              <LogicButton variant="and">Logic Button</LogicButton>
+              <ToggleButton>Toggle Button</ToggleButton>
+            </ButtonGroup>
+          </Section>
+
+          <Section title="Collections">
+            <ActionMenu>
+              <Item>Action Menu Item 1</Item>
+              <Item>Action Menu Item 2</Item>
+              <Item>Action Menu Item 3</Item>
+            </ActionMenu>
+            <ListBox width="size-2400" aria-label="Listbox">
+              <Item>ListBox Item 1</Item>
+              <Item>ListBox Item 2</Item>
+              <Item>ListBox Item 3</Item>
+            </ListBox>
+            <ListView
+              selectionMode="multiple"
+              aria-label="Static ListView items example"
+              maxWidth="size-6000"
+            >
+              <Item>Adobe Photoshop</Item>
+              <Item>Adobe InDesign</Item>
+              <Item>Adobe AfterEffects</Item>
+              <Item>Adobe Illustrator</Item>
+              <Item>Adobe Lightroom</Item>
+            </ListView>
+            <MenuTrigger>
+              <ActionButton>Menu</ActionButton>
+              <Menu onAction={(key) => alert(key)}>
+                <Item key="cut">Cut</Item>
+                <Item key="copy">Copy</Item>
+                <Item key="paste">Paste</Item>
+                <Item key="replace">Replace</Item>
+              </Menu>
+            </MenuTrigger>
+            <MenuTrigger>
+              <ActionButton>Menu Trigger</ActionButton>
+              <Menu>
+                <Item>Cut</Item>
+                <Item>Copy</Item>
+                <Item>Paste</Item>
+              </Menu>
+            </MenuTrigger>
+            <TableView
+              aria-label="Example table with static contents"
+              selectionMode="multiple"
+            >
+              <TableHeader>
+                <Column>Name</Column>
+                <Column>Type</Column>
+                <Column align="end">Date Modified</Column>
+              </TableHeader>
+              <TableBody>
+                <Row>
+                  <Cell>Games</Cell>
+                  <Cell>File folder</Cell>
+                  <Cell>6/7/2020</Cell>
+                </Row>
+                <Row>
+                  <Cell>Program Files</Cell>
+                  <Cell>File folder</Cell>
+                  <Cell>4/7/2021</Cell>
+                </Row>
+                <Row>
+                  <Cell>bootmgr</Cell>
+                  <Cell>System file</Cell>
+                  <Cell>11/20/2010</Cell>
+                </Row>
+                <Row>
+                  <Cell>log.txt</Cell>
+                  <Cell>Text Document</Cell>
+                  <Cell>1/18/2016</Cell>
+                </Row>
+              </TableBody>
+            </TableView>
+          </Section>
+
+          <Section title="Color">
+            <ColorArea defaultValue="#7f0000" />
+            <ColorField label="Primary Color" />
```

Comment:
In the color section we have ColorField rendering inline while the others are on newlines, seems by design.
![image](https://user-images.githubusercontent.com/309540/194613988-199562e1-f7e3-45cf-b38c-fdf09c65410b.png)